app/bt/ExposureNotificationContext.tsx

```diff
@@ -33,6 +33,8 @@ const ExposureNotificationsProvider = ({
   const [deviceStatus, setDeviceStatus] = useState<DeviceStatus>(initialStatus);
 
   useEffect(() => {
+    getCurrentENPermissionsStatus();
```

Comment:
Maybe this should happen outside of the abstraction? 
So GPS and BT initialize their permission states?

Reply:
good call - I think `ExposureNotificationsContext` could be replaced by a context that handles "permissions" more generally, the way you have it set up on the gps side. We'll be adding `Notifications` permissions on the BT side soon, so that could live in `bt/PermissionsContext` too 🤔